src/DETHRACE/common/world.c

```diff
@@ -2806,8 +2809,21 @@ void LoadTrack(char* pFile_name, tTrack_spec* pTrack_spec, tRace_info* pRace_inf
             strcat(str, ".MAT");
             material = LoadSingleMaterial(&gTrack_storage_space, str);
             pRace_info->material_modifiers[i].skid_mark_material = material;
+#if defined(DETHRACE_FIX_BUGS)
+            skid_mark_cnt++;
+#endif
         }
     }
+#if defined(DETHRACE_FIX_BUGS)
+    /* Display skidmarks even if the race has no specified skidmark material. */
+    if (!skid_mark_cnt && num_materials) {
```

Comment:
why can't we just check `if (pRace_info->material_modifiers[0].skid_mark_material == NULL) { ... `?

Reply:
We could, but my idea is to only apply this fix if none of the materials specify skidmarks, not just the first one.

Reply:
Ok, got it